helx/charts/roger/.gitignore

```diff
@@ -0,0 +1 @@
+charts/
```

Comment:
Don't think this is needed.

Reply:
The charts/ dir  on helm dependency build/update will download zip files for airflow and redis charts , didn't want to add those in in git

Reply:
At first I was thinking that only the root .gitignore would be used, so I learned something new.  Now I'm thinking this shouldn't be here b/c when I build the charts the zip files will need to be checked in for the devops helm repo.

Reply:
This makes sense, maybe we could do a rule like ignore all zips except the ones you have built under (`docs/charts/`), at the root .gitignore ? 
Also when you said they shouldn't be here is

Reply:
I was referring to roger/.gitignore.  I believe we can use something like "!*.zip" or "!*.tar.gz" in roger/.gitignore.  I haven't really needed an ignore for .zip or .tar.gz files so far.